genomix/genomix-pregelix/src/main/java/edu/uci/ics/genomix/pregelix/testhelper/BridgeAddVertex.java

```
@@ -41,7 +42,12 @@ public void initVertex() {
         if (kmerSize == -1) {
             kmerSize = 3; // Integer.parseInt(getContext().getConfiguration().get(GenomixJobConf.KMER_LENGTH));
         }
-        GenomixJobConf.setGlobalStaticConstants(getContext().getConfiguration());
+        try {
```

if not, you can always wrap it in some other error, e.g., RuntimeException() or something. Better to error somehow than to fail silently.